scene/gui/control.cpp

```diff
@@ -2961,7 +2961,7 @@ void Control::end_bulk_theme_override() {
 // Internationalization.
 
 TypedArray<Vector3i> Control::structured_text_parser(TextServer::StructuredTextParser p_parser_type, const Array &p_args, const String &p_text) const {
-	ERR_READ_THREAD_GUARD_V(TypedArray<Vector3i>());
+	//ERR_READ_THREAD_GUARD_V(TypedArray<Vector3i>());
```

Comment:
This function is working only with its arguments, so the default handler is thread safe, but I'm not sure what's the safe way to make `GDVIRTUAL_CALL` override from a thread.